src/main/java/View/Api/Get/POST_Sensor_Get.java

```diff
@@ -1,11 +1,11 @@
-package Database.Interaction.Servlets.Get;
```

Comment:
kedze kazda z Api/Get a Api/Post obsahuje servlety pre rovnake entity, vidis ze cele to speje k tomu aby sa tie pary zlucili do 1 servletu. Tak to aj byva, Entita a cely CRUD co mozno najviac na jednom mieste